hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.js

```diff
@@ -1,26 +1,151 @@
-        </div>
+            <div className={styles.inventory}>
+                <Drawer
+                    className={styles.inventoryFilterDrawer}
+                    variant="permanent"
+                    variant={mobileWidth ? "permanent" : "temporary"}
+                    anchor="left"
+                    open={mobileOpen}
+                    onClose={toggleFilter}
+                >
+                    <InventoryFilter
+                        categories={inventoryCategories}
+                        applyFilter={applyFilter}
+                        removeFilter={removeFilter}
+                    />
+                </Drawer>
+                <div className={styles.inventoryBody}>
+                    <Typography variant="h1">Hardware Inventory</Typography>
+
+                    <div className={styles.inventoryBodyToolbar}>
+                        <Grid
+                            direction="row"
+                            alignItems="center"
+                            className={styles.inventoryBodyToolbarDiv}
+                            container
+                        >
+                            <TextField
+                                className={styles.inventoryBodyToolbarSearch}
+                                id="search-input"
+                                label="Search items"
+                                variant="outlined"
+                                size="small"
+                            />
+                            <IconButton
+                                color="primary"
+                                aria-label="Search"
+                                variant="contained"
+                            >
+                                <SearchIcon />
+                            </IconButton>
+                        </Grid>
+
+                        <Divider
+                            orientation="vertical"
+                            className={styles.inventoryBodyToolbarDivider}
+                            flexItem
+                        />
+
+                        <Grid
+                            direction="row"
+                            justify="space-between"
+                            alignItems="center"
+                            className={styles.inventoryBodyToolbarDiv}
+                            container
+                        >
+                            <Hidden implementation="css" mdUp>
+                                <Button
+                                    aria-label="Orders"
+                                    startIcon={<FilterListIcon />}
+                                    onClick={toggleFilter}
+                                >
+                                    Filter
+                                </Button>
+                            </Hidden>
+
+                            <div className={styles.inventoryBodyToolbarRefresh}>
+                                <Typography variant="body2">123 items.</Typography>
```

Comment:
The most minor thing ever, but the fullstop feels weird. Can change this around when the page gets connected